applications/api-cat/src/test/java/no/acat/spec/converters/SwaggerJsonSpecConverterTest.java

```diff
@@ -0,0 +1,38 @@
+package no.acat.spec.converters;
+
+import io.swagger.v3.oas.models.OpenAPI;
+import no.acat.utils.IOUtillity;
+import no.dcat.shared.testcategories.UnitTest;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.mockito.MockitoAnnotations;
+
+import java.io.IOException;
+
+@Category(UnitTest.class)
+public class SwaggerJsonSpecConverterTest {
+
+  String spec;
+
+  @Before
+  public void setup() throws IOException {
+    MockitoAnnotations.initMocks(this);
+    spec = IOUtillity.getStringOutputFromFile("swagger-raw-enhet-api.json");
+  }
+
+  @Test
+  public void test_if_canConvert() {
+
+    boolean convert = SwaggerJsonSpecConverter.canConvert(spec);
```

Comment:
Igjen, hver test skulle ha klar fire steg
1)setup
2)operation
3)check
4)teardown

Hvis mulig, unngå shared state mellom test.

I din test her skulle du ha to linjer:
spec=loadResource...
assertTrue(canConvert(...)

Reply:
ok